ocis-pkg/tracing/config.go

```diff
@@ -8,7 +8,7 @@ type ConfigConverter interface {
 // Tracing defines the available tracing configuration.
 type Config struct {
 	Enabled   bool   `yaml:"enabled" env:"OCIS_TRACING_ENABLED" desc:"Activates tracing." introductionVersion:"pre5.0"`
-	Type      string `yaml:"type" env:"OCIS_TRACING_TYPE" desc:"The type of tracing. Defaults to \"\", which is the same as \"jaeger\". Allowed tracing types are \"jaeger\" and \"\" as of now." introductionVersion:"pre5.0" introductionVersion:"pre5.0"`
+	Type      string `yaml:"type" env:"OCIS_TRACING_TYPE" desc:"The type of tracing. Defaults to \"\", which is the same as \"jaeger\". Allowed tracing types are \"jaeger\", \"otlp\" and \"\" as of now." introductionVersion:"pre5.0" introductionVersion:"pre5.0"`
```

Comment:
```suggestion
	Type      string `yaml:"type" env:"OCIS_TRACING_TYPE" desc:"The type of tracing. Defaults to '', which is the same as 'jaeger'. Allowed tracing types are 'jaeger', 'otlp' and '' as of now." introductionVersion:"pre5.0" introductionVersion:"pre5.0"`
```
For the sake that we everywhere else use single quotes instead of double quotes. This for sure was just a leftover from ancient times :sweat_smile: